nni/tools/nnictl/command_utils.py

```diff
@@ -65,6 +77,72 @@ def install_requirements_command(requirements_path):
     return call(_get_pip_install() + ["-r", requirements_path], shell=False)
 
 
+def _wait_till_process_killed(pid, timeout):
+    keyboard_interrupted = False
+    time_count = 0
+    # Usually, a process is killed very quickly.
+    # This little nap will save 1 second.
+    time.sleep(0.01)
+    while True:
+        try:
+            # Implementation of waiting
+            while time_count < timeout:
+                pid_running = _check_pid_running(pid)
```

Comment:
Ah, this time is your fault.
Use `process.is_running()`.
https://psutil.readthedocs.io/en/latest/#psutil.Process.is_running